src/js/controllers/walletHome.js

```diff
@@ -46,18 +46,6 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
     self.resetForm();
   });
 
-  var disableOnlineListener = $rootScope.$on('Local/Online', function() {
-    // This is needed then the apps go to sleep
-    self.bindTouchDown();
-  });
-
-  var disableResumeListener = $rootScope.$on('Local/Resume', function() {
```

Comment:
I think this needs to stay. If you put the up to sleep, and the resume, the 'fast' buttons (menu button, "tap event on press") need to be rebinded.